pkgs/samples/analysis_options.yaml

```diff
@@ -6,4 +6,8 @@ analyzer:
 
 linter:
   rules:
+    - directives_ordering
+    - prefer_final_in_for_each
```

Comment:
I'm not sure about this one and the next one. These samples are user facing; the code should probably closely follow the lints from package:lints / package:flutter_lints.

Reply:
I.e., I think it's OK if this particular analysis options file differs from the others in this repo.

Reply:
That makes sense to me. I removed the final ones and left the others so that we can keep the samples a bit more consistent.